temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -85,6 +89,12 @@ public static WorkflowServiceStubsOptions getDefaultInstance() {
   /** Frequency at which connection backoff is going to be reset */
   private final Duration connectionBackoffResetFrequency;
 
+  /**
+   * Frequency at which grpc connection channel will be moved into an idle state, triggering a new
+   * connection to the temporal frontend host.
+   */
+  private final Duration enterIdleChannelStateFrequency;
```

Comment:
What about a more user-friendly name? Something like "grpcReconnectFrequency"?

Reply:
I like it, let me rename.